Stratis.Bitcoin/Features/BlockStore/LoopSteps/DownloadBlockStep.cs

```diff
@@ -13,23 +13,24 @@ namespace Stratis.Bitcoin.Features.BlockStore.LoopSteps
     ///<list>
     /// There are two operations:
     ///     <item>1: <see cref="BlockStoreInnerStepFindBlocks"/> to ask the block puller to download the blocks.</item>
-    ///     <item>2: <see cref="BlockStoreInnerStepDownloadBlocks"/> to persist the blocks in a batch to the <see cref="BlockRepository"/>.</item>
+    ///     <item>2: <see cref="BlockStoreInnerStepReadBlocks"/> to persist the blocks in a batch to the <see cref="BlockRepository"/>.</item>
     /// </list>
     /// </para> 
     /// <para>
-    /// After a "Stop" condition is found the FindBlocksTask will be removed from 
-    /// the <see cref="BlockStoreInnerStepContext.InnerSteps"/> and only the 
-    /// <see cref="BlockStoreInnerStepDownloadBlocks"/> will continue to execute until the <see cref="BlockStoreInnerStepContext.DownloadStack"/> is empty.
+    /// After a "Stop" condition is found the AskBlocksTask will be removed from
+    /// <see cref="BlockStoreInnerStepContext.InnerSteps"/> and only the 
+    /// <see cref="BlockStoreInnerStepReadBlocks"/> task will continue to execute 
+    /// until the DownloadStack is empty.
```

Comment:
`<see>` here for DownloadStack, and for AskBlockTask above, but again, it seems that AskBlocksTask does not exist in the code. Was it renamed?